lib/modules/manager/gomod/extract.ts

```diff
@@ -62,7 +66,14 @@ export function extractPackageFile(
       const line = lines[lineNumber];
       const goVer = line.startsWith('go ') ? line.replace('go ', '') : null;
       if (goVer && semver.validRange(goVer)) {
-        const dep = getGoDep(lineNumber, goVer);
+        const dep = getGoDep(lineNumber, goVer, 'go-mod-directive');
+        deps.push(dep);
```

Comment:
```suggestion
        deps.push(dep);
        continue;
```
i think we don't need to do any futher check son this line